contribute thomas

thomasraffray opened the pull request.

thomasraffray committed:
upload projet

thomasraffray committed:
upload

thomasraffray committed:
Create .gitignore

thomasraffray committed:
Update slnx.sqlite

thomasraffray committed:
Merge branch 'damienmarchal:main' into main

thomasraffray committed:
thomas

thomasraffray committed:
Merge branch 'main' of https://github.com/thomasraffray/junia-2024

thomasraffray committed:
Delete out directory

thomasraffray committed:
Delete .vs directory

thomasraffray committed:
Thomas Soleil

thomasraffray committed:
Merge branch 'main' of https://github.com/thomasraffray/junia-2024

thomasraffray committed:
Merge remote-tracking branch 'upstream/main'

thomasraffray committed:
bug environnement

damienmarchal commented on src/walle-lib/environnement.hpp:
```diff
@@ -2,13 +2,13 @@
+    Environment(double left=-1, double right=-1, double top=-1, double bottom=1);
 
-    double left() { return -1; }
+    double left(); //{ return -1; }
```
this was a quick & dirty test. Shouldn't be committed or merged

damienmarchal commented on src/walle-lib/robot.hpp:
```diff
@@ -27,7 +27,7 @@ class Robot
 
     /// retourne la capacity de la batterie.
-    virtual double get_battery_capacity() = 0;
+    virtual double get_battery_capacity() {};
```
This lines probably produce a warning message because there is no returned value. This can crash or not depending on the compiler.

damienmarchal commented:
Hi, 
thanks for the PR, 
eager to see the continuation of the it.

thomasraffray committed:
projet C++ robot planteur

Contribution Thomas RAFFRAY
Création d'un environnement, affichage du robot et algorithme de plantage d'arbres
